.vscode/settings.json

```diff
@@ -18,7 +18,8 @@
     "app",
     "ts",
     "column",
-    "board"
+    "board",
+    "radicle-interface"
```

Comment:
I'd just call that `iframe` or something smaller 'cause I've found that long contexts make it hard to fit the commit description. But you do you.

Reply:
I'll take your advice, changed!